CMakeLists.txt

```diff
@@ -197,3 +197,8 @@ file(WRITE ${CMAKE_BINARY_DIR}/dali/__init__.py "")
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/lint.cmake)
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Install.cmake)
+
+add_library(DALI::dali ALIAS dali)
+add_library(DALI::dali_core ALIAS dali_core)
+add_library(DALI::dali_kernels ALIAS dali_kernels)
+add_library(DALI::dali_operators ALIAS dali_operators)
```

Comment:
Is it possible to additionally create one target, that puts together all these four?

One to rule them all ;)

Reply:
I do not think so, at least I did not find one